content/doc/book/blueocean/creating-pipelines.adoc

```diff
@@ -1,135 +1,290 @@
+  account or organization) from which to build your Pipeline project. +
+  *Tip:* If your list of repositories is long, you can filter this list using
+  the *Search* option.
+[.boxshadow]
+image:blueocean/creating-pipelines/choose-a-repository.png['Choose a
+repository',width=70%]
+. Click *Create Pipeline*. +
+  Blue Ocean will then scan your local repository's branches for a `Jenkinsfile`
+  and will commence a Pipeline run for each branch containing a `Jenkinsfile`.
+  If Blue Ocean cannot find any `Jenkinsfile`, you will be prompted to begin
+  creating one through the <<pipeline-editor#,Blue Ocean Pipeline editor>> (by
+  clicking *Create Pipeline* again). +
+  *Note:* Under the hood, a Pipeline project created through Blue Ocean is
+  actually "multibranch Pipeline". Therefore, Jenkins looks for the presence of
+  at least one Jenkinsfile in any branch of your repository.
+
+[NOTE]
+====
+When you register the Github access token for your current/logged in Jenkins
+user in Blue Ocean, Jenkins registers this credential with the following details
+for this Jenkins user:
+
+* *Domain*: `blueocean-github-domain`
+* *ID*: `github`
+* *Name*: `<jenkins-username>/****** (GitHub Access Token)`
+====
+
+////
+=== From a repository on GitHub Enterprise
+////
+
+=== From a repository on Bitbucket Cloud
+
+To create your Pipeline project directly from a repository on BitBucket Cloud,
+click the *BitBucket Cloud* button under *Where do you store your code?*
 
-[[github-auto-discover]]
-==== Auto-discover Pipelines
+[.boxshadow]
+image:blueocean/creating-pipelines/where-do-you-store-your-code.png['Where do
+you store your code',width=70%]
 
-Selecting "Auto-discover Pipelines" scans all repositories belonging to the selected owner,
-and will create a Pipeline for each branch containing a "Jenkinsfile" in the root folder.
+In the *Connect to Bitbucket* section, enter your Bitbucket email address and
+password into the *Username* and *Password* fields, respectively. +
+If you previously configured Blue Ocean to connect to Bitbucket with your email
+address and password, Blue Ocean takes you directly to the
+<<choose-your-bitbucket-account-team-and-repository,choosing your Bitbucket
+account/team and repository>> steps below.
 
 [.boxshadow]
-image:blueocean/creating-pipelines/github-auto-discover.png['Discover All
-Pipelines for Owner',width=60%]
-
-This option is useful for adding Pipelines for all the repositories in an organization,
-when those repositories already have `Jenkinsfile` entries in them.
-Repositories that do not contain `Jenkinsfile` entries are ignored.
-To create a new `Jenkinsfile` in a single repository that does not have one, use the
-"<<github-new-pipeline, New Pipeline>>" option instead.
+image:blueocean/creating-pipelines/connect-to-bitbucket.png['Connect to
+Bitbucket',width=70%]
+
+Click *Connect* and your current/logged in Jenkins user wil now have access to
```

Comment:
s/wil/will/

Reply:
D'Oh ... Don't know why my spellchecker in Atom is failing - probably needs a reboot.